src/test/java/az/edu/strangers/FamilyTest.java

```diff
@@ -18,10 +23,15 @@ void setUp() {
         mother = new Human("Jane", "Karleone", 1975);
         father = new Human("Vito", "Karleone", 1973);
         family = new Family(mother, father);
-        pet = new Pet(Species.DOG, "Rock", 5, 75, new String[]{"eat", "sleep"});
+        Set<String> habits = new HashSet<>();
+        habits.add("Sleep");
+        habits.add("Run");
+        pet = new Pet(Species.DOG, "Rock", 5, 75, new String[] {"eat" , "sleep"});
```

Comment:
Why array?

Reply:
Test classes will be reverted back, i will handle them.